ga4gh/backend.py

```diff
@@ -218,6 +276,12 @@ def addDataset(self, dataset):
         self._datasetNameMap[dataset.getLocalId()] = dataset
         self._datasetIds.append(id_)
 
+    def addOntologies(self, ontologies):
+        """
+        Adds ontologies to this backend.
```

Comment:
Hmm, maybe I'm misunderstanding here though. What's an ontology? It would be nice to give a little bit of detail about what the parameter is here.